test-configs.yaml

```diff
@@ -1117,26 +1117,15 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
-  r8a7795-salvator-x: &r8a7795-salvator-x
+  r8a77960-ulcb: &r8a77960-ulcb
```

Comment:
Tested OK with stable 5.8: https://lava.collabora.co.uk/scheduler/job/2776404
However I believe the dtb name needs to be manually overridden for `r8a7796-m3ulcb` in older kernels.